person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def of_age?
+    @age >= 18
+  end
+  private :of_age?
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+  public :can_use_services?
+end
```

Comment:
- [ ] As mentioned by the previous reviewer, just as required in this milestone, kindly ensure your person class inherits the properties of the Nameable. 
- [ ] Also, ensure the Person class has a method correct_name that returns the name attribute as required.

### Project Requirements
![project-rqt](https://github.com/MICHAELKITH/oop-library/assets/26597432/7e964df7-9bc5-4141-a876-b9ff6db9f639)

Reply:
Not Resolved ✖️ 

![image](https://github.com/MICHAELKITH/oop-library/assets/38088818/7e72c7d7-0520-4298-b114-352d5add3959)